app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java

```diff
@@ -80,4 +80,40 @@ public static void onResponse(int responseCode) {
         }
     }
 
+    /**
+     * Last WindowsSetting constructor values. Values are checked for change to reduce log spam.
+     */
+    private static int lastAp, lastAnchorHorizontalPos, lastAnchorVerticalPos;
+    private static boolean lastZ1, lastZ2;
+
+    /**
+     * Injection point.
+     */
+    public static int[] getSubtitleWindowSettingsOverride(int ap, int anchorHorizontalPos, int anchorVerticalPos,
+                                                         boolean z1, boolean z2) {
+        int[] override = {ap, anchorHorizontalPos, anchorVerticalPos};
+
+        if (SettingsEnum.DEBUG.getBoolean()) {
+            if (ap != lastAp || anchorHorizontalPos != lastAnchorHorizontalPos
+                    || anchorVerticalPos != lastAnchorVerticalPos
+                    || z1 != lastZ1 || z2 != lastZ2) {
+                LogHelper.printDebug(() -> "SubtitleWindowSettings ap:" + ap + " anchorHorizontalPos:" + anchorHorizontalPos
+                        + " anchorVerticalPos:" + anchorVerticalPos + " z1:" + z1 + " z2:" + z2);
+                lastAp = ap;
+                lastAnchorHorizontalPos = anchorHorizontalPos;
+                lastAnchorVerticalPos = anchorVerticalPos;
+                lastZ1 = z1;
+                lastZ2 = z2;
+            }
+        }
+
+        if (SettingsEnum.SIGNATURE_SPOOFING.getBoolean() && !PlayerType.getCurrent().isNoneOrHidden()) {
+            override[0] = 34;
```

Comment:
Please document those magic values for future reference, it is unclear what these values represent.